docs/source/tutorials/ScadaInterfaceExample.ipynb

```diff
@@ -0,0 +1,615 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2021-06-17T14:54:08.411307Z",
+     "start_time": "2021-06-17T14:53:13.435259Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "import straxen\n",
+    "import matplotlib.pyplot as plt"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Tutorial SCADA-Python :\n",
+    "\n",
+    "In this notebook we show the basic functionalities of our SCADA interface. As a first step we have to initialize the interface. Please note that this is only possible for XENON members who are working on Midway. The straxen client will request a security token from our slowcontrol webservice. This token is only valid 3 hrs."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 23,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2021-06-17T15:31:49.625746Z",
+     "start_time": "2021-06-17T15:31:49.155245Z"
+    }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Received token, the token is valid for 3 hrs.\n",
+      " from 17.06. 15:31:49 UTC\n",
+      " till 17.06. 18:31:49 UTC\n",
+      "We will automatically refresh the token for you :). Have a nice day and a fruitful analysis!\n"
+     ]
+    }
+   ],
+   "source": [
+    "sc = straxen.SCADAInterface()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "After getting the token we can query some parameters. Lets try to get some data for a certain time range and set of different parameters. The parameters can be specified via a dictionary which takes as values the Historian database names of the required parameter and some short name as key. Let us query as an example some different parameters:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 24,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2021-06-17T15:31:49.729703Z",
+     "start_time": "2021-06-17T15:31:49.725985Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "parameters = {'para1': 'XE1T.GEN_HEINZVMON.PI',\n",
+    "              'para2': 'XE1T.CTPC.Board06.Chan011.VMon',\n",
+    "              'para2': 'XE1T.CRY_TE101_TCRYOBOTT_AI.PI',\n",
+    "              'para3': 'XE1T.CRY_PT101_PCHAMBER_AI.PI',\n",
+    "              'para4': 'XE1T.GEN_CE911_SLM1_HMON.PI',\n",
+    "              'para5': 'XE1T.CRY_FCV104FMON.PI',\n",
+    "             }"
```

Comment:
Thanks Daniel, can we add a link or reference where to get these parameter conventions from?

Reply:
Yes, good idea. I asked SC group.